tests/test-common.js

```diff
@@ -1133,6 +1181,40 @@ describe('JSON-LD Signatures', function() {
         });
       });
 
+      it.skip('signs a local document using a custom signer', async () => {
```

Comment:
This and the test above are skipped... why? Can we enable them?

Reply:
Oh whoops, I forgot about these. They're disabled cause I changed the API, these no longer apply. But I meant to add equivalent tests and remove these. Thanks!